lib/matplotlib/font_manager.py

```diff
@@ -1341,6 +1344,12 @@ def _findfont_cached(self, prop, fontext, directory, fallback_to_default,
                 _log.warning(
                     'findfont: Font family %s not found. Falling back to %s.',
                     prop.get_family(), self.defaultFamily[fontext])
+                for family in map(str.lower, prop.get_family()):
+                    if family in font_family_aliases:
+                        _log.warning(
+                            "findfont: Font family %r not found because none "
+                            "of the following families were found: %s",
```

Comment:
Is this families, or fonts?

Reply:
These are all families (https://developer.mozilla.org/en-US/docs/Web/CSS/font-family), a "font" includes not only the family but also the size, etc.  I did slightly reword the message to refer to (e.g.) "sans" as a "generic family", though (again following CSS terminology).